build.sbt

```diff
@@ -148,6 +148,13 @@ lazy val site = project
   )
   .dependsOn(kernel, githubActions, noPublish)
 
+lazy val unidoc = project
+  .in(file("unidoc"))
+  .enablePlugins(TypelevelUnidocPlugin)
+  .settings(
+    name := "sbt-typelevel-unidoc"
+  )
```

Comment:
This is how you set it up. Choice of "unidoc" for the module name is arbitrary—it would be good to establish a best practice.

Reply:
Actually: this doesn't might not even need a separate module. In theory you could enable the `TypelevelUnidocPlugin` on your core module and it would replace its API docs artifact with the unidoc artifact. Not entirely sure if this would work—could be playing with fire.